### ISSUE TITLE
[Bug]: Incomplete translation when text contains single quote

### ISSUE THREAD

**Issue by apiaget** [laravel-idea/plugin#914]
### Bug description

When generating a translation for a text containing a single quote, I expect that the lang file contains the text in full.

### Plugin version

8.0.3.233

### Operating system

Windows

### Steps to reproduce

In `SiteController.php`
```php
public function index() {
    $text = "Nouvelle offre d'emploi"; // We select the text and click ALT + Ins. I will use the following translation key : "admin.test"
    return view('index');
}
```

In `lang/fr/admin.php`
```php
<?php
return [
    'test' => 'Nouvelle offre d',
]; 
```

The translation generation works fine in blade templates. The bug occurs only in Controllers, Enums, Models, etc.

**Comment by adelf** [comment]
Thanks. Will be fixed.

**Comment by adelf** [comment]
Fixed in 8.0.4. Thank you.
